sui/src/benchmark.rs

```diff
@@ -142,8 +141,8 @@ fn run_throughout_microbench(
             .block_on(async move { send_tx_chunks(txes, network_client, connections).await });
 
         let _: Vec<_> = resp
-            .par_iter()
-            .map(|q| check_transaction_response(deserialize_message(&(q.as_ref().unwrap())[..])))
+            .into_par_iter()
```

Comment:
Are you sure this makes a big difference? There is no crypto going on here right?

Reply:
No the main reason is here https://github.com/MystenLabs/sui/pull/1753/files#diff-d034a711191f7478e5cf0da617e12fc6836d4245f23e622d2c71048f33b64567R50-R71